plugins/modules/provision_workflow_manager.py

```diff
@@ -789,13 +932,28 @@ class instance for further use.
                         return self
 
         elif device_type == "wireless":
-            response = self.dnac_apply['exec'](
-                family="wireless",
-                function="provision",
-                op_modifies=True,
-                params=self.want.get("prov_params"),
-            )
-            self.log("Wireless provisioning response collected from 'provision' API is: {0}".format(response), "DEBUG")
+            try:
+                response = self.dnac_apply['exec'](
+                    family="wireless",
+                    function="provision",
+                    op_modifies=True,
+                    params={"payload": self.want.get("prov_params")}
+                )
+                self.log("Wireless provisioning response collected from 'provision' API is: {0}".format(str(response)), "DEBUG")
+                execution_id = response.get("executionId")
+                provision_info = self.get_execution_status_wireless(execution_id=execution_id)
+                self.result["changed"] = True
+                self.result['msg'] = "Wireless device got provisioned successfully"
```

Comment:
Is it possible to add the name/ip of which wireless device has been successfully provisioned?

Reply:
Addressed